index.html

```diff
@@ -30,21 +30,21 @@
 
                     </div>
                     <div>
-                        <input class="inner-input" type="" name="" placeholder="Phone number">
+                        <input input class="other-inputs" type="" name="" placeholder="Phone number">
                     </div>
                     <div class="label">
                         <label>Enter your email (required)</label>
 
                     </div>
                     <div>
-                        <input type="" name="" placeholder="Email">
+                        <input input class="other-inputs" type="" name="" placeholder="Email">
                     </div>
                     <div class="label">
                         <label>Enter a password (required)</label>
 
                     </div>
                     <div>
-                        <input type="" name="" placeholder="Password">
+                        <input input class="other-inputs" type="" name="" placeholder="Password">
```

Comment:
"input" word appears twice. Change it so that it appears once

Reply:
Also, you need to give that field a type of password to add some minimal HTML5 validation as opposed to leaving it blank